frontend/components/panels/DesktopClassPanel.js

```diff
@@ -190,6 +191,7 @@ export default class DesktopClassPanel extends BaseClassPanel {
                 {courseAttr}
               </div>
             </Collapsible>
+            {feeString}
```

Comment:
Can you include a screenshot? On github, if you have the image in your clipboard you can just hit ctrl-v in the text box and it will include it in the comment

Reply:
![image](https://user-images.githubusercontent.com/33675688/55502938-a393f800-561b-11e9-8090-96c24a08a5e0.png)

Reply:
Super cool, can we put a dollar sign in front of that? and remove the '.00'?

Reply:
And maybe it would be worth adding a tooltip on top of it that says "You must pay $651 extra to take this class". Maybe it would be worth adding a "exclamation" icon to the left of Music Lab fee to make it stand out more?

idk I would have to play around in Sketch a bit and see what looks clear and good.

Reply:
Do we know that fees will always be an integer values?

Reply:
It would be kept as a "number" type in JS - we would show the decimals if it has any

Reply:
Yeah, Javascript just auto turns "651.00" -> 651. Since the number is being stored as a Number type, it'll auto truncate the 0's when they're not needed (when it's an integer value). Should be cool ish!

Reply:
no toolbar/exclamation yet... anyone have cool idea for em?